libs/execution_context/ebpf_program.h

```diff
@@ -414,6 +417,39 @@ extern "C"
     size_t
     ebpf_program_get_state_index();
 
+    /**
+     * @brief Get whether the program information provider supports the context header.
+     *
+     * @param[in] program Pointer to the program object.
+     *
+     * @return true when the program supports the context header.
+     * @return false when the program does not support the context header.
```

Comment:
```suggestion
     * @retval true The program supports the context header.
     * @retval false The program does not support the context header.
```
`return` should only appear once, but `retval` can appear once per return value.